dev/packages/alpha/nginx/0.0.3/manifest.yml

```diff
@@ -0,0 +1,51 @@
+format_version: 1.0.0
+name: nginx
+title: Nginx
+version: 0.0.3
+license: basic
+description: Nginx Integration
+type: integration
+categories:
+- logs
+- metrics
+release: ga
```

Comment:
probably more like beta

Reply:
Ok, I can change this, but it means that we should update also MySQL?

Reply:
Fixed for this package